CleverTapSDK/CleverTap.m

```diff
@@ -85,7 +85,7 @@
 NSString *const kQUEUE_NAME_EVENTS = @"events";
 NSString *const kQUEUE_NAME_NOTIFICATIONS = @"notifications";
 
-NSString *const kHANDSHAKE_URL = @"https://eu1.clevertap-prod.com/hello";
+NSString *const kHANDSHAKE_URL = @"https://clevertap-prod.com/hello";
```

Comment:
@akashvercetti `clevertap-prod.com` is already there in the Constants file. Can we follow good coding practices and reuse the constant so that we change it only at once place if needed in the future?